src/sentry/lang/javascript/processor.py

```diff
@@ -381,6 +391,21 @@ def fetch_file(url, project=None, release=None, dist=None, allow_scraping=True):
     return result
 
 
+def get_max_age(headers):
+    cache_control = 'cache-control'
+    max_age = CACHE_CONTROL_MIN
+
+    if has_header(headers, cache_control):
+        match = re.search(CACHE_CONTROL_RE, headers[cache_control])
```

Comment:
Since `CACHE_CONTROL_RE` is already a compiled regex object, you can/should just do `CACHE_CONTROL_RE.search` instead of `re.search`.